### PR TITLE
Add fireside chat session.

### PR DESCRIPTION
**Pull request by hotchemi**

### CODE REVIEW

**Comment by konifar** [comment]
:eyes:

File: app/src/main/res/raw/sessions_ar.json [hunk 1]
```diff
@@ -1119,14 +1119,14 @@
...
-    "description": "[نظرة عامة] وقال التطبيقات التي يمكن أن يتم تسليم إبلاغ مباشرة على الإعلام الجهاز ن لمعدلات أعلى مفتوحة من مجلة البريد الإلكتروني، هي ميزة قوية جدا حتى عند إجراء الاحتفاظ للن المستخدمين غير نشط.\n ومع ذلك، اعتمادا على محتويات الإخطار المفرط أو إخطار أو متوقف إخطار التطبيق، فإنه سوف يكون أيضا السبب الذي يمكن أن يكون لحذف ن أسوأ التطبيق القضية. مناقشة إخطار الأمثل يعني من التطبيق الأصلي في ن ن هذا الإعلان، وكان في الواقع الفعلي من أجل زيادة إخطار جهود معدلات مفتوحة، ن وأود أن نتحدث عن كيفية تنفيذ ذلك.\n ن [موضوع] الذين يقومون بتشغيل التطبيق في ن · مبتدئين تطوير الروبوت إلى وسيط المستخدم ن · الأعمال ن ن [الكلمة] ن · GCM ن · Notidfication ن · ملابس ن محلي إخطار ن · إخطار المنبثقة",
+    "title": "Fireside chat",
+    "description": "المطورين ملخص من \ ن التخطيط لحالات الطوارئ! DroidKaigi التطبيق الرسمي ويجمع، وأنا أقول فيما يتعلق \ إنتاج ن قصة سرية. \ هل معجزة ما حدث إلى n بكل الوسائل استنفدت تقريبا كان المشهد ... لا تفوت!",
     "speaker": {
       "id": 43,
-      "name": "cooky_ch",
-      "image_url": "https://avatars0.githubusercontent.com/u/3312886?v=3&s=400.jpg",
+      "name": "DroidKaigi developers",
+      "image_url": "",
```
Comment (konifar) [inline app/src/main/res/raw/sessions_ar.json]: Suggestion.
How about DroidKaigi logo url?

Comment (hotchemi) [inline app/src/main/res/raw/sessions_ar.json]: Do you know the proper size one? Please tell me!

Comment (konifar) [inline app/src/main/res/raw/sessions_ar.json]: I guess any size is OK.

Comment (hotchemi) [inline app/src/main/res/raw/sessions_ar.json]: Added!

**Comment by konifar** [comment]
LGTM! Thanks!!

File: app/src/main/res/raw/sessions_en.json [hunk 2]
```diff
@@ -1119,14 +1119,14 @@
...
-    "title": "「通知オフ」にされない為に。ネイティブアプリの最適な通知について",
-    "description": "【概要】\nデバイスに直接お知らせを配信できるアプリの通知は、メルマガよりも開封率が高いと言われ、\n非アクティブなユーザーに対するリテンションを行う際にも非常に強力な機能です。\nしかし、過度な通知や通知の内容によってはそのアプリの通知をオフにされたり、\n最悪の場合アプリを削除されてしまう原因にもなります。\n\n本発表ではネイティブアプリの最適な通知手段について考察し、実際に通知の開封率を上げる為に効果的だった取り組みと、\nその実装方法について話したいと思います。\n\n【対象者】\n・Android開発 初心者〜中級者\n・業務でアプリを運用している方\n\n【キーワード】\n・GCM\n・Notidfication\n・Wear\n・ローカル通知\n・ポップアップ通知",
+    "title": "Fireside chat",
+    "description": "【概要】\n緊急企画! DroidKaigi official appの開発者達が一堂に会し、\n制作秘話に関して語ります。\n万策尽きかけた現場に起こった奇跡とは…乞うご期待!",
     "speaker": {
       "id": 43,
-      "name": "cooky_ch",
-      "image_url": "https://avatars0.githubusercontent.com/u/3312886?v=3&s=400.jpg",
+      "name": "DroidKaigi developers",
+      "image_url": "https://raw.githubusercontent.com/konifar/droidkaigi2016/master/app/src/main/res/mipmap-mdpi/ic_launcher.png",
```
Comment (konifar) [inline app/src/main/res/raw/sessions_en.json]: Sorry, `xdpi` seems better.

2016-02-18_23_32_39_png

Comment (konifar) [inline app/src/main/res/raw/sessions_en.json]: 2016-02-18_23_32_50_png

**Comment by konifar** [comment]
Merge it!

**Pull request merged**